baselines/fedmix/fedmix/femnist/README.md

```diff
@@ -0,0 +1,34 @@
+# FEMNIST Dataset
```

Comment:
I think it's better to remove the entire `femnist` directory and instead tell those that want to run your `FedMix` baseline to clone the original `LEAF` repository. Then provide some instructions on how to download and pre-process `FEMNIST`. You can use for inspiration how another contributor did this [FedMeta](https://github.com/adap/flower/pull/2438). Please add the instructions in your `README.md`.